### ISSUE TITLE
Build warning and render error

### ISSUE THREAD

**Issue by Hyphon** [antfu/unplugin-vue2-script-setup#95]
when i run `npm run build` in https://github.com/antfu/unplugin-vue2-script-setup/tree/main/examples/vue-cli

and there is a warning:
![image](https://user-images.githubusercontent.com/11419600/147799292-eb094c26-0dcd-4267-a0fb-203326d1da7d.png)

then i visit the web. there is a error in console:
![image](https://user-images.githubusercontent.com/11419600/147799306-07f418a9-3958-4168-a54d-7e81b69e77d1.png)

SystemInfo:
```
System:
    OS: Linux 5.10 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (3) x64 Intel(R) Core(TM) i5-9400 CPU @ 2.90GHz
    Memory: 2.66 GB / 7.77 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.13.1 - /home/packages/node/bin/node
    Yarn: 1.22.17 - /home/packages/node/bin/yarn
    npm: 8.1.2 - /home/packages/node/bin/npm
  npmPackages:
    @vue/compiler-core: ^3.2.26 => 3.2.26 
    @vue/compiler-dom: ^3.2.26 => 3.2.26 
    @vue/composition-api: ^1.4.3 => 1.4.3 
    @vue/ref-transform: ^3.2.24 => 3.2.24 
    @vue/shared: ^3.2.26 => 3.2.26 
```

**Comment by xiaoxiangmoe** [comment]
fixed in https://github.com/antfu/unplugin-vue2-script-setup/pull/96

**Comment by Hyphon** [comment]
> fixed in #96

Could you plz release this fix ?
